scrapy/utils/misc.py

```diff
@@ -1,6 +1,8 @@
 """Helper functions which don't fit anywhere else"""
+import os
 import re
 import hashlib
+from contextlib import contextmanager
```

Comment:
:lipstick: Please, keep the alphabetical order.

Reply:
Is this the wanted order? I don't know which alphabetic order should be aplied since re and hashlib were also not in alphabetic order. I know made it start with the normal import in alphabetic order followed by the from imports in alphabetic order

Reply:
I see, I did not realize it was common to have `from … import` lines after regular `import` lines. Ignore my comment :slightly_smiling_face: